expression/simple_rewriter.go

```diff
@@ -105,7 +116,32 @@ func RewriteSimpleExprWithSchema(ctx sessionctx.Context, expr ast.ExprNode, sche
 	return rewriter.pop(), nil
 }
 
+// FindColName finds the column name from []*namingForMySQLProtocol.
+func FindColName(names []*NamingForMySQLProtocol, astCol *ast.ColumnName) (int, error) {
+	dbName, tblName, colName := astCol.Schema, astCol.Table, astCol.Name
+	idx := -1
+	for i, name := range names {
+		if (dbName.L == "" || dbName.L == name.DBName.L) &&
+			(tblName.L == "" || tblName.L == name.TblName.L) &&
+			(colName.L == name.ColName.L) {
+			if idx == -1 {
+				idx = i
+			} else {
+				return -1, errors.Errorf("Column %s is ambiguous", name.String())
```

Comment:
It's better to tell the users which column it duplicates to help they address this error quickly.

Reply:
Current error msg is the same with MySQL.

Reply:
How about using `ErrNonUniq`? It's defined in the parser package.

Reply:
got